hack/python-sdk/gen-sdk.sh

```diff
@@ -21,31 +21,47 @@ set -o pipefail
 SWAGGER_JAR_URL="https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/5.1.0/openapi-generator-cli-5.1.0.jar"
 SWAGGER_CODEGEN_JAR="hack/python-sdk/openapi-generator-cli.jar"
 SWAGGER_CODEGEN_CONF="hack/python-sdk/swagger_config.json"
-SWAGGER_CODEGEN_FILE="pkg/apis/kubeflow/v1/swagger.json"
+SWAGGER_V1_CODEGEN_FILE="pkg/apis/kubeflow/v1/swagger.json"
+SWAGGER_V2_CODEGEN_FILE="v2/pkg/apis/kubeflow/v2beta1/swagger.json"
 SDK_OUTPUT_PATH="sdk/python"
 
 if [ -z "${GOPATH:-}" ]; then
     export GOPATH=$(go env GOPATH)
 fi
 
-# Backup existing openapi_generated.go
+# Backup existing v1 openapi_generated.go
 mv pkg/apis/kubeflow/v1/openapi_generated.go pkg/apis/kubeflow/v1/openapi_generated.go.backup
 
-echo "Generating OpenAPI specification ..."
+echo "Generating V1 OpenAPI specification ..."
 openapi-gen --input-dirs github.com/kubeflow/mpi-operator/pkg/apis/kubeflow/v1,github.com/kubeflow/common/pkg/apis/common/v1 --output-package github.com/kubeflow/mpi-operator/pkg/apis/kubeflow/v1 --go-header-file hack/boilerplate/boilerplate.go.txt
 
-echo "Generating swagger file ..."
-go run hack/python-sdk/main.go 0.1 > ${SWAGGER_CODEGEN_FILE}
+echo "Generating V1 swagger file ..."
+go run hack/python-sdk/main.go V1 v1 > ${SWAGGER_V1_CODEGEN_FILE}
+
+# Backup existing v2 openapi_generated.go
+mv v2/pkg/apis/kubeflow/v2beta1/openapi_generated.go v2/pkg/apis/kubeflow/v2beta1/openapi_generated.go.backup
+
+echo "Generating V2 OpenAPI specification ..."
+openapi-gen --input-dirs github.com/kubeflow/mpi-operator/v2/pkg/apis/kubeflow/v2beta1,github.com/kubeflow/common/pkg/apis/common/v1 --output-package github.com/kubeflow/mpi-operator/v2/pkg/apis/kubeflow/v2beta1 --go-header-file hack/boilerplate/boilerplate.go.txt
+
+echo "Generating V2 swagger file ..."
+go run hack/python-sdk/main.go V2 v2beta1 > ${SWAGGER_V2_CODEGEN_FILE}
```

Comment:
To clarify my other point, why can't this be

`go run hack/python-sdk/main.go v2beta1 > ${SWAGGER_V2_CODEGEN_FILE}`

Reply:
Understand now, will update.